### PR TITLE
Mythfrontend

### PR DESCRIPTION
**Pull request by jhoyt4**
*If there’s a checkbox you can’t complete for any reason, that's okay, just explain in detail why you weren’t able to do so.*

After making all changes to the cask:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` reports no offenses.
- [x] The commit message includes the cask’s name and version.

Additionally, **if adding a new cask**:

- [x] Named the cask according to the [token reference].
- [x] `brew cask install {{cask_file}}` worked successfully.
- [x] `brew cask uninstall {{cask_file}}` worked successfully.
- [x] Checked there are no [open pull requests] for the same cask.
- [x] Checked the cask was not already refused in [closed issues].
- [x] Checked the cask is submitted to [the correct repo].

[token reference]: https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md
[open pull requests]: https://github.com/caskroom/homebrew-cask/pulls
[closed issues]: https://github.com/caskroom/homebrew-cask/issues?q=is%3Aissue+is%3Aclosed
[the correct repo]: https://github.com/caskroom/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask

### CODE REVIEW

File: Casks/mythfrontend.rb [hunk 1]
```diff
@@ -0,0 +1,11 @@
+cask 'mythfrontend' do
+  version '0.28-52'
+  sha256 '2e0c4de89d1ab91a86f5d6ea3ec0a60654412e01f54db28c679cc8e30bf62140'
+
+  # downloads.sourceforge.net was verified as official when first introduced to the cask
+  url 'https://downloads.sourceforge.net/mythtvformacosx/MythFrontend-0.28-intel-10.9-v0.28-52-ge6a60f7.dmg'
+  name 'MythFrontend'
```
Comment (vitorgalvao) [inline Casks/mythfrontend.rb]: Please add an [`appcast`](https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/appcast.md).

**Review by vitorgalvao — CHANGES_REQUESTED** [review]
Please add an [`appcast`](https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/appcast.md).

**Comment by vitorgalvao** [comment]
Tip for the future: don’t type `check`, use a `x` (edited, see the result).

**Comment by jhoyt4** [comment]
appcast added.  All steps reverified.

**Comment by vitorgalvao** [comment]
You’ve added a bunch of unrelated commits.

File: Casks/mythfrontend.rb [hunk 2]
```diff
@@ -0,0 +1,14 @@
+cask 'mythfrontend' do
+  version '0.28-52'
+  sha256 '2e0c4de89d1ab91a86f5d6ea3ec0a60654412e01f54db28c679cc8e30bf62140'
+
+  # downloads.sourceforge.net was verified as official when first introduced to the cask
+  url 'https://downloads.sourceforge.net/mythtvformacosx/MythFrontend-0.28-intel-10.9-v0.28-52-ge6a60f7.dmg'
+  appcast 'https://sourceforge.net/projects/mythtvformacosx/rss?path=/',
+           checkpoint: 'ab4861cf53ab2a9d006aec31ed89378ad242bef1199eb1864436076f3e39c583'
+
```
Comment (vitorgalvao) [inline Casks/mythfrontend.rb]: This does not pass `style`.

**Review by vitorgalvao — CHANGES_REQUESTED** [review]

File: Casks/mythfrontend.rb (hunk 2 above)
Comment (jhoyt4) [inline Casks/mythfrontend.rb]: just close / cancel the pull request.  I'm struggling to learn both git and github at the same time and have no clue how to clean up my mess.

Comment (vitorgalvao) [inline Casks/mythfrontend.rb]: Very well, don’t worry, it’s complicated for everyone at the start.

Care to try again in a new PR?

Comment (jhoyt4) [inline Casks/mythfrontend.rb]: Will do.

**Pull request closed without merging**